### PR TITLE
Adding additional optional ci jobs

### PR DESCRIPTION
**Pull request by coreydaley**
- e2e-azure
- e2e-azure-operator
- e2e-gcp
- e2e-gcp-operator

### CODE REVIEW

File: ci-operator/config/openshift/cluster-image-registry-operator/openshift-cluster-image-registry-operator-master.yaml [hunk 1]
```diff
@@ -49,3 +49,27 @@ tests:
   openshift_installer:
     cluster_profile: aws
     upgrade: true
+- as: e2e-gcp
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
+    cluster_profile: gcp
+    upgrade: false
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/cluster-image-registry-operator/openshift-cluster-image-registry-operator-master.yaml]: omit this

**Comment by bparees** [comment]
you're going to need to get @stevekuznetsov's team to approver this, i don't think they want us running all these jobs all the time on all PRs anywhere...they probably need to be configured as run_always: false, and you can explicitly run them on PRs where you are concerned about the changes, for now.

**Comment by coreydaley** [comment]
@stevekuznetsov ptal
I have made all of the new jobs optional and to not run every time (except for verify)

**Comment by coreydaley** [comment]
@stevekuznetsov bump

**Pull request merged**